pkgs/top-level/all-packages.nix

```diff
@@ -301,6 +301,8 @@ with pkgs;
       inherit kernel rootModules allowMissing;
     };
 
+  mkShell = callPackage ../build-supports/shell-builder.nix { };
```

Comment:
This should be `build-support`, not `build-supports`.

Reply:
I'd suggest moving it into `../build-supports/shell-builder/default.nix` anyway. Quiet a few derivations there are already single file.

Reply:
fixed